Intermittent Docker build failure

On PR #5998 from dependabot, there was initially [a failure in the publish_storybooks task in CircleCI](https://app.circleci.com/pipelines/github/mozilla/experimenter/10405/workflows/0a45a07a-34ff-4b4f-ad09-ff659d7e6f9e/jobs/30732) at the `$(COMPOSE_TEST)` command:

```
#10 [dev  3/24] COPY --from=file-loader /app/bin/ /app/bin/
#10 sha256:51e1f58d8611759830220f3481166eeff621e7acab42209c23de0e6338475294
#10 ERROR: extract requires an applier
------
 > [dev  3/24] COPY --from=file-loader /app/bin/ /app/bin/:
------
failed to compute cache key: extract requires an applier
make: *** [Makefile:66: build_test] Error 1

Exited with code exit status 2

CircleCI received exit code 2
```

On rerun, [this passed without error](https://app.circleci.com/pipelines/github/mozilla/experimenter/10405/workflows/c5609859-0366-4b5e-b9d3-a3e96ceaf9f0/jobs/30754):

```
#10 [dev  3/24] COPY --from=file-loader /app/bin/ /app/bin/
#10 sha256:76ebc66382640f51ce1b4da59a2cc593fab77586b88483837dd6f95d6598db22
#10 CACHED
```

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-1516)

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -2,7 +2,6 @@ version: 2.1
 jobs:
   check:
     machine:
-      docker_layer_caching: true
       image: ubuntu-2004:202104-01 # Ubuntu 20.04, Docker v20.10.6, Docker Compose v1.29.1
     resource_class: large
     working_directory: ~/experimenter
@@ -22,7 +21,6 @@ jobs:
 
   publish_storybooks:
     machine:
-      docker_layer_caching: true
       image: ubuntu-2004:202104-01 # Ubuntu 20.04, Docker v20.10.6, Docker Compose v1.29.1
     resource_class: medium
     working_directory: ~/experimenter
@@ -42,7 +40,6 @@ jobs:
 
   integration_legacy:
     machine:
-      docker_layer_caching: true
       image: ubuntu-2004:202104-01 # Ubuntu 20.04, Docker v20.10.6, Docker Compose v1.29.1
     resource_class: xlarge
     working_directory: ~/experimenter
@@ -64,7 +61,6 @@ jobs:
 
   integration_nimbus:
     machine:
-      docker_layer_caching: true
       image: ubuntu-2004:202104-01 # Ubuntu 20.04, Docker v20.10.6, Docker Compose v1.29.1
     resource_class: xlarge
     working_directory: ~/experimenter
@@ -87,7 +83,6 @@ jobs:
   deploy:
     working_directory: ~/experimenter
     machine:
-      docker_layer_caching: true
       image: ubuntu-2004:202104-01 # Ubuntu 20.04, Docker v20.10.6, Docker Compose v1.29.1
     steps:
       - checkout

